### DISCUSSION TITLE
Users and Customers in NexoPOS 5.x

### DISCUSSION THREAD

**Discussion by swurzinger** [Blair2004/NexoPOS#1343] [General]
In NexoPOS both users and customers are internally very similar and stored in the same database table. So each customer is basically also a user (and vice-versa).
Still, there are different UIs for creating/updating/managing users and customers and different APIs.

What was the reasoning/rationale for unifying users and customers?
Why hasn't it been done in the UI or in APIs?
Are there any further changes planned regarding users/customers?

**Comment by Blair2004** [comment]
We plan NexoPOS to be an eCommerce solution as well on the long run. Customers will then be able to login and manage their account. We didn't wanted to create 2 authentication on the same system for achieving the same feature. More over, we wanted to allow any type of user (even users with custom roles) to be able to login, manage their account (including making sales etc).

**Reply by swurzinger** [reply]
makes sense, thanks!
